stubs/responses.pyi

```diff
@@ -15,9 +15,6 @@ def add(
     status: Optional[int] = None,
     json: Optional[JsonDict] = None,
 ): ...
-
-T = TypeVar("T")
-
-def activate(T) -> T: ...
+def activate(Callable) -> Callable: ...
```

Comment:
what is this change doing? Is it related to updating the versions of python and urllib?

Reply:
When updating the version of the `mypy` typechecker, it started complaining about the previous type-hint.  We use this only as a decorator of test functions for handling response mocking. This satisfies it and also seems reasonable.